lua/terrortown/lang/en.lua

```diff
@@ -1418,7 +1418,7 @@ L.spawneditor_desc = "Used to place weapon, ammo and player spawns in the world.
 L.spawneditor_place = "Place spawn"
 L.spawneditor_remove = "Remove spawn"
 L.spawneditor_change = "Change spawn type (hold [SHIFT] to reverse)"
-L.spawneditor_ammo_edit = "Hold to edit amount of autospawning ammo on weapon spawns"
+L.spawneditor_ammo_edit = "Hold on weapon spawn to edit autospawning ammo"
```

Comment:
comment this out in the other languages then?

Reply:
I don't think it is that important. I only wanted to shorten in in German and English because the text was way too long. If someone else was bothered by it in their language, they can edit it. But it is not game-breaking